app/api/domains/osu.py

```diff
@@ -521,7 +521,7 @@ async def osuSearchHandler(
         result = await resp.json()
 
         if USING_CHIMU:
-            if result["code"] != 0:
+            if result["code"] != 200:
                 return b"-1\nFailed to retrieve data from the beatmap mirror."
```

Comment:
traduzir essa linha que passou despercebido tb

Reply:
se o status do request HTTP não for 0 (código não existente), falhar a função. (sempre ira falhar)
para
se o status do request HTTP não for 200 (código que significa sucesso), falhar a função. (somente ira falhar se request for mal sucedido)

Reply:
sim mano, tô falando da mensagem de falha mesmo

Reply:
ata, btw acho q vc tem permissão de editar

Reply:
to no celular irmão, não posso fazer isso agora..

Reply:
d boa, só queria confirmar. Já traduzi